tests/components/broadlink/test_helpers.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for Broadlink helper functions."""
+import pytest
+import voluptuous as vol
+
+from homeassistant.components.broadlink.helpers import data_packet, mac_address
+
+
+async def test_padding(hass):
+    """Verify that non padding strings are allowed."""
+    assert data_packet("Jg") == b"&"
+    assert data_packet("Jg=") == b"&"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6042836)